src/kayenta/reducers/data.ts

```diff
@@ -94,7 +94,7 @@ const executions = combineReducers<IExecutionsState>({
       [Actions.LOAD_EXECUTIONS_SUCCESS]: () => AsyncRequestState.Fulfilled,
       [Actions.LOAD_EXECUTIONS_FAILURE]: () => AsyncRequestState.Failed,
     },
-    AsyncRequestState.Requesting,
+    AsyncRequestState.Fulfilled,
```

Comment:
we'll start in a "Fulfilled" state so that we load the table component and let it activate the data source; otherwise, it never activates and loads its data